File: godfather/roles/mafia/vanilla_mafia.py
from godfather.roles.base import Role
from godfather.roles.mixins import MafiaMember

DESCRIPTION = 'You have no powers.'


class VanillaMafia(MafiaMember, Role):
    """
    A regular mafia without any special powers.

    - Win Condition: Kill anyone who will not submit to the mafia.

    + Abilities:
    + If the Goon dies, you will be given the final say over nightkill selection.
    """
    name = 'Vanilla Mafia'
    description = DESCRIPTION

File: src/lib/mafia/roles/mafia/Vanilla_Mafia.ts
import MafiaRole from '@mafia/mixins/MafiaRole';
import Role from '@mafia/structures/Role';
import type Player from '@mafia/structures/Player';

class VanillaMafia extends Role {
	public name = 'Vanilla Mafia';

	public constructor(player: Player) {
		super(player);
		this.description = this.game.t('roles/mafia:vanillaMafiaDescription');
	}
}

VanillaMafia.aliases = [...VanillaMafia.aliases, 'Mafia Vanilla'];

export default MafiaRole(VanillaMafia);
